### ISSUE TITLE
Accessibility problems with interactive elements

### ISSUE THREAD

**Issue by pmarsh-scottlogic** [gsproston-scottlogic/prompt-injection#523]
This ticket was born out of the investigation in #308 

1. Defence configuration inputs should have linked label elements. E.g. `max message length` should become a `<label>` with the `for` property set to the id of the input. [example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). Remove any `aria-label`s from the inputs. ([more information](https://dequeuniversity.com/rules/axe/4.8/label?application=AxeChrome))
![Image](https://github.com/ScottLogic/prompt-injection/assets/118171430/8e8f812d-719e-401b-9563-1fd732bfc8e3)
2. Add aria-label to view documents fowards and backwards buttons so that they are screen-readable.
![Image](https://github.com/ScottLogic/prompt-injection/assets/118171430/a82926e1-afda-4fac-86fd-67d1613219d4)
3. Short mission info button has unclear screen reader text. add an aria-label which will override the internal text for screen readers.
![Image](https://github.com/ScottLogic/prompt-injection/assets/118171430/fe33d3a3-b365-49c6-ab3c-bab0500a99d8)
